canonical-web-and-design/canonicalwebteam.blog
# Add feeds endpoints for authors, groups and topics

## DONE
Add missing feeds endpoints and build the feed ourselves instead of redirecting to WP.

## TODO
- [x] Have a better look at URL generation
- [x] Update feed titles for author, groups and topics.

## QA
- Pull this branch.
- Run `poetry build`
- In your fork of `www.ubuntu.com` pull this branch https://github.com/canonical-web-and-design/ubuntu.com/pull/5693/files.
- In `www.ubuntu.com` requirements.txt swap the `canonicalwebteam.blog` requirement for the local version you cloned on the first step(use the tar.gz in the `dist` folder of your local copy of `canonicalwebteam.blog`).
- Run without docker and try to append `/feed` to different blog url's (groups, topics and authors) and make sure it returns similar content to what you see on the actual page.
- Do the same on a flask project like `jp.ubuntu.com` (no need to clone any specific branch, just swap the requirement)
